scripts/api.py

```diff
@@ -98,7 +98,12 @@ async def txt2img(
         controlnet_processor_res: int = Body(512, title='Controlnet Processor Res'),
         controlnet_threshold_a: int = Body(64, title='Controlnet Threshold a'),
         controlnet_threshold_b: int = Body(64, title='Controlnet Threshold b'),
-        controlnet_guidance: float = Body(1.0, title='ControlNet Guidance Strength'),
+        guidance_strength:float = Body(1.0, title='Guidance Strength'),
```

Comment:
Guidance strength may be confused with other options/extensions, how about ControlNet Guidance Strength?